coin_change_problem/CoinChange.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Implementation of famous dynamic programming problem
+ * that aims to find out the maximum number of ways in
+ * which a value can be achieved using some fixed valued
+ * coins.
+ *
+ * In the implementation, the time complexity is O(mn)
+ * and extra space required is O(n).
+ */
+import java.util.Arrays;
+/*
+ * coin: Array containing value of coins
+ * m: Number of types of coins
+ * n: Value to find the change for
+ */
+public class CoinChange {
+	public static int coinChangeProblem(int[] C, int m, int n) {
```

Comment:
Is sure does, but for the user it is redundant. At this point, it is just an extra parameter which is not at all needed.